policy/modules/system/systemd.te

```diff
@@ -1264,6 +1285,21 @@ seutil_read_file_contexts(systemd_sessions_t)
 
 systemd_log_parse_environment(systemd_sessions_t)
 
+########################################
+#
+# sysctl local policy
+#
+
+dontaudit systemd_sysctl_t self:capability sys_ptrace;
+
+kernel_read_kernel_sysctls(systemd_sysctl_t)
+kernel_request_load_module(systemd_sysctl_t)
+kernel_rw_all_sysctls(systemd_sysctl_t)
```

Comment:
This seems unnecessary, why not let it run in `initrc_t`, which is where non-systemd systems will typically be setting sysctls?

Reply:
In my testing I was getting several AVCs where `initrc_t` was not permitted to set some types of sysctls. I thought to just grant that access to `initrc_t` but felt that a new domain made more sense, and that's also how the Fedora policy handled this. I'm fine with reverting to the former however.